.scalafmt.conf

```diff
@@ -1,16 +1,18 @@
-version = "2.4.2"
+version = "2.6.4"
 maxColumn = 120
-align = most
+align.preset = most
+align.multiline = false
 continuationIndent.defnSite = 2
 assumeStandardLibraryStripMargin = true
 docstrings = JavaDoc
 lineEndings = preserve
 includeCurlyBraceInSelectChains = false
-danglingParentheses = true
+danglingParentheses.preset = true
 spaces {
   inImportCurlyBraces = true
 }
 optIn.annotationNewlines = true
+newlines.alwaysBeforeMultilineDef = false
```

Comment:
A lot of changes could be related to more strict formatting of expressions like this:

```scala
// before
def suite = suite("My suite)(
  testM("My test") { 
    // ...
  }
)

// after
def suite =
  suite("My suite)(
    testM("My test") { 
      // ...
    }
  )
```

I see that first style is preferred in this repo, so disabled it